src/components/Nav/SecondNav/SecondNav.js

```diff
@@ -20,9 +33,20 @@ const SecondNav = () => {
               );
             })}
           </NavDropdownList>
-          <SearchBar>
-            <Input />
-            <SearchIcon src="/images/Nav/loupe.png" />
+
+          <SearchBar
+            onSubmit={e => {
+              e.preventDefault();
+            }}
+          >
+            <Input
+              id="search"
+              value={userInput}
+              type="text"
+              onChange={handleSearch}
+              onKeyPress={e => onSubmit(e)}
+            />
```

Comment:
onSubmit 함수에서 매개변수로 이벤트 객체 받으시고, `e.preventDefault()` 해주시면 됩니다!

```suggestion
          <SearchBar>
            <Input
              id="search"
              value={userInput}
              type="text"
              onChange={handleSearch}
              onKeyPress={onSubmit}
            />
```